lib/scan.ts

```diff
@@ -153,8 +153,36 @@ export function getExcludedPatterns(
 
   const config: DotSnykConfig = dotSnyk.parse(policyFilePath);
 
-  return [
-    policyFilePath,
-    ...dotSnyk.toAbsolutePaths(projectRoot, config?.exclude?.global),
-  ];
+  const today = new Date();
+
+  const paths: readonly string[] | undefined = config?.exclude?.global
+    ?.filter((item) => {
+      if (typeof item === 'object') {
+        const key = Object.keys(item)[0];
+
+        if (!key) {
+          return false;
+        }
+
```

Comment:
here you could extract `item[key]` and assign it to a const and reuse it. It's used four times after this line.